### ISSUE TITLE
[BUG] HTTP request failed, statusCode: 404,

### ISSUE THREAD

**Issue by zekkontro** [Hexer10/youtube_explode_dart#101]
I get a 404 error with thumbnails every time I pull a playlist. How can I solve the error?

```
  Future getMusic() async {
    videos = await yt.playlists.getVideos(widget.playlistId).toList();

    return videos;
  }
```
```
 StreamBuilder<Object>(
                stream: getMusic().asStream(),
                builder: (context, snapshot) { .... }
```
HTTP request failed, statusCode: 404, https://i9.ytimg.com/s_p/RDCLAK5uy_l63FFr2xGnXGVPfOWTNtAirWzsmZW_fBU/mqdefault.jpg?sqp=CICj5YAGir7X7AMGCIvJ_-kF&rs=AOn4CLAQvN4FuKMsLs__xCBzBofSbZqxpw&v=1564468363

 - Enviroment: Flutter o Dart VM
 - Version  1.22.6
 - YoutubeExplode Version 1.7.5

**Comment by zekkontro** [comment]
Other than that, video is not searched. Keeps you waiting

**Comment by Hexer10** [comment]
Fixed in `1.8.0`
